tglib/certpool.go

```diff
@@ -0,0 +1,12 @@
+// +build !windows
+
+package tglib
+
+import (
+	"crypto/x509"
+)
+
+// GetSystemCertPool gets the system cert pool via Go libraries
+func GetSystemCertPool() (*x509.CertPool, error) {
```

Comment:
Can call it SystemCerPool() so we can easily replace later when they finally support it in the stdlib?